src/components/organisms/Footer/Links.tsx

```diff
@@ -0,0 +1,51 @@
+import { Link } from 'gatsby'
+import React, { ReactElement } from 'react'
+import { useGdprMetadata } from '../../../hooks/useGdprMetadata'
+import { useSiteMetadata } from '../../../hooks/useSiteMetadata'
+import { useUserPreferences } from '../../../providers/UserPreferences'
+import Button from '../../atoms/Button'
+import styles from './Links.module.css'
+
+export default function Links(): ReactElement {
+  const { appConfig, footer } = useSiteMetadata()
+  const { setShowPPC, privacyPolicySlug } = useUserPreferences()
+  const { content } = footer
+
+  const cookies = useGdprMetadata()
+
+  return (
+    <div className={styles.container}>
+      {content?.map((section, i) => (
+        <div key={i} className={styles.section}>
+          <p className={styles.title}>{section.title}</p>
+          <div className={styles.links}>
+            {section.links.map((e, i) => (
+              <Link key={`${e.name}-${i}`} to={e.link}>
+                {e.name}
+              </Link>
+            ))}
+          </div>
+        </div>
+      ))}
+      <div className={styles.section}>
+        <p className={styles.title}>PRIVACY</p>
```

Comment:
suggestion: could actually also be moved into the content `.json` to allow chaning the title more easily

Reply:
Yes, it's like this because the whole privacy section is a little bit weirdly built with all these exceptions but I can add this to the json.